bvm/ballerina-config/src/main/java/org/ballerinalang/config/ConfigProcessor.java

```diff
@@ -95,15 +99,15 @@ private static void lookUpVariables(Map<String, Object> configFileEntries) {
                         try {
                             configFileEntries.put(entry.getKey(), Long.parseLong(envVarVal));
                         } catch (NumberFormatException e) {
-                            throw new IllegalArgumentException(
-                                    "invalid int value received from environment: " + envVarVal);
+                            errMsgs.add("invalid int value received from environment to '" +
```

Comment:
Instead of using a list, can't we use a StringBuilder itself? We can check the length of string builder later to see if it should be printed. StringBuilder appending is faster than string concatenation like this.

Reply:
Not sure there's much to gain here by using a StringBuilder?

Reply:
I meant since we are anyway using a StringBuilder later what if we use it earlier on. Here it will reduce two string concatenations.